.github/workflows/conflict_labeler.yml

```diff
@@ -0,0 +1,24 @@
+name: Merge Conflict Labeler
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request_target:
+  issue_comment:
+
+permissions: {}
+
+jobs:
+  label:
+    name: Labeling
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Sonarr/Sonarr' }}
+    steps:
+      - name: Apply label
+        uses: eps1lon/actions-label-merge-conflict@v3
+        if: ${{ github.event_name == 'push' || github.event_name == 'pull_request_target'}}
+        with:
+          dirtyLabel: 'merge conflict'
+          commentOnDirty: 'This pull request has merge conflicts. Please resolve the conflicts so the PR can be successfully reviewed and merged.'
```

Comment:
Is it possible to exclude the comment? I don't think it is necessary as authors will get notifications when the label is added.

Reply:
I don't think people get the notification when their PR is labeled, the only notification you get on a PR is comment and commit

Reply:
So you would prefer no comment at all ?